core/src/main/java/org/openedx/core/module/db/DownloadModelEntity.kt

```diff
@@ -22,7 +24,13 @@ data class DownloadModelEntity(
     @ColumnInfo("downloadedState")
     val downloadedState: String,
     @ColumnInfo("progress")
-    val progress: Float?
+    val progress: Float?,
+    @ColumnInfo("transcriptUrls")
+    val transcriptUrls: String,
+    @ColumnInfo("transcriptPaths")
+    val transcriptPaths: String,
```

Comment:
The general idea is that it won't be a problem because a `String` in Room maps to the `TEXT` type in SQLite, which can store up to 2 GB of data, provided the device has sufficient resources.

[Limits in SQLite - Point 1](https://www.sqlite.org/limits.html)